src/master-playlist-controller.js

```diff
@@ -804,4 +814,42 @@ export default class MasterPlaylistController extends videojs.EventTarget {
       }
     });
   }
+
+  updateCues_(media) {
+    if (!this.useCueTags_ || !media.segments) {
+      return;
+    }
+
+    while (this.cueTagsTrack_.cues.length) {
+      this.cueTagsTrack_.removeCue(this.cueTagsTrack_.cues[0]);
+    }
+
+    let startTime;
+    let endTime = 0;
+
+    for (let i = 0; i < media.segments.length; i++) {
+      let segment = media.segments[i];
+
+      startTime = endTime;
+      endTime = startTime + segment.duration;
```

Comment:
ID3 uses a duration of "0" (start time and end time are equal).

The goal is that we want to trigger these cue points once we cross between segments (for us, we are considering it the beginning of the segment the tag precedes). We only need it to trigger the cue change and remain active long enough for a listener of the event to grab the cue.

@gkatsev is the recommendation for 0.5 seconds the minimum for it remaining active long enough for a user to retrieve and make use of?